server/routes/profile.js

```diff
@@ -1,40 +1,52 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const multer = require('multer');
-const path = require('path');
-const protect = require('../middleware/auth');
+const multer = require("multer");
+const path = require("path");
+const protect = require("../middleware/auth");
 const {
   editProfile,
   loadProfile,
-  retriveImgUrls
-} = require('../controllers/profile');
+  retrieveImgUrls,
+  retrieveAvatarUrl,
+  getAvatar,
+  deleteProfilePhoto
+} = require("../controllers/profile");
 
 const fileStorageEngine = multer.diskStorage({
   destination: function (req, file, cb) {
-    cb(null, './uploads');
+    cb(null, "./uploads");
   },
   filename: function (req, file, cb) {
-    const fileName = `${Date.now()} ${path.extname(file.originalname)}`;
+    const fileName = `${Date.now()}${path.extname(file.originalname)}`;
     cb(null, fileName);
   }
 });
 
-const upload = multer({ 
+const upload = multer({
   storage: fileStorageEngine,
   fileFilter: function (req, file, callback) {
-    var ext = path.extname(file.originalname);
-    if(ext !== '.png' && ext !== '.jpg' && ext !== '.gif' && ext !== '.jpeg') {
-        return callback(new Error('Only images are allowed'))
-    };
-    callback(null, true)
-}, });
+    const ext = path.extname(file.originalname);
+    if (ext !== ".png" && ext !== ".jpg" && ext !== ".gif" && ext !== ".jpeg") {
+      return callback(new Error("Only images are allowed"));
+    }
```

Comment:
`.webp` is also a format that may be uploaded too. We should also include that in our conditional. Not commonly used for profile photos, but it's possible a user could upload that and we want to make sure it still is allowed 🙂